webpack.config.js

```diff
@@ -33,77 +29,64 @@ module.exports = {
         test: /\.scss$/,
         include: path.resolve(__dirname, "css"),
         use: [
-          MiniCssExtractPlugin.loader,
-          {
-            loader: "css-loader",
-            options: {
-              url: false,
-            },
-          },
+          "css-loader",
           "sass-loader",
         ],
       },
       {
-        test: /\.html$/,
-        use: [
-          {
-            loader: "html-loader",
-            options: { minimize: false },
-          },
-        ],
+        test: /\.(png|jpe?g|ico|svg)$/,
+        type: "asset/resource",
+        generator: {
+          filename: "img/[name].[hash:8][ext]",
+        },
+      },
+      {
+        test: /\.(woff2?)$/,
+        type: "asset/resource",
+        generator: {
+          filename: "fonts/[name][ext]",
+        },
       },
     ],
   },
+  // enable HMR with live reload
   devServer: {
     port: 8000,
-    static: false,
-    hot: false,
-    watchFiles: ["html_output/**/*.html", "css/*.scss", "scripts/*.js"],
-    devMiddleware: {
-      writeToDisk: true,
+    open: true,
```

Comment:
Lets keep `open: false` - personal preference

Reply:
Yes, I will change to false. For me self I keep always `open: false,`. There was just for easy start of  demonstration.

I will rebase the PR with actual `main` branch to resolve all possible conflicts.